Maths/FindLcm.js

```diff
@@ -26,16 +26,16 @@ const findLcm = (num1, num2) => {
   let maxNum
   let lcm
   // Check to see whether num1 or num2 is larger.
-  if (num1 > num2) {
-    maxNum = num1
-  } else {
-    maxNum = num2
-  }
+  maxNum = Math.max(num1, num2)
   lcm = maxNum
 
   while (true) {
     if (lcm % num1 === 0 && lcm % num2 === 0) break
-    lcm += maxNum
+    if (num1 > num2) {
```

Comment:
Why did you change this?

Reply:
It was by mistake. I was confused that's why.